doc/classes/ClassDB.xml

```diff
@@ -192,6 +192,15 @@
 				Returns whether this [param class] is enabled or not.
 			</description>
 		</method>
+		<method name="is_class_enum_bitfield" qualifiers="const">
+			<return type="bool" />
+			<param index="0" name="class" type="StringName" />
+			<param index="1" name="enum" type="StringName" />
+			<param index="2" name="no_inheritance" type="bool" default="false" />
+			<description>
+				Returns whether the [param enum] is defined as a bitfield in [param class] or its ancestor classes.
```

Comment:
I'd say it should still follow that style suggested as this doesn't clarify the option, meant to fix the other cases later, this should be made clear from the start

Reply:
So would you agree on this description @AThousandShips 
```
Returns whether [param class] or its ancestor classes has an enum called [param enum] that is a bitfield.
```

Reply:
It should mention that `no_inheritance` controls things, like my suggestion following the general form of those cases